cluster-management/setup/mounting-storage/index.md

```diff
@@ -64,6 +64,42 @@ coreos:
 
 Notice that we're starting both units at the same time and using the power of systemd to work out the dependencies for us. In this case, `var-lib-docker.mount` requires `format-ephemeral.service`, ensuring that our storage will always be formatted before it is mounted. Docker will refuse to start otherwise.
 
+## Creating and Mounting a btrfs volume file
```

Comment:
Mixed title case here. Should it be:

`Creating and Mounting a btrfs Volume File`

?